moped-database/metadata/tables.yaml

```diff
@@ -3140,6 +3076,13 @@
           value_from_env: MOPED_API_APIKEY
         - name: MOPED_API_EVENT_NAME
           value: activity_log
+      cleanup_config:
+        batch_size: 10000
+        clean_invocation_logs: false
+        clear_older_than: 168
+        paused: true
+        schedule: 0 0 * * *
+        timeout: 60
```

Comment:
This feature was apparently introduced in `v2.13` but is an enterprise-only feature. Too bad! We'll have to keep running our cron job.

https://github.com/hasura/graphql-engine/releases/tag/v2.13.0

Reply:
will this break somehow including this in here since we dont have enterprise edition? or are you manifesting it to be included in a newer release?

Reply:
oh maybe its because my activity log edits happened to a previous version of graphql-engine